plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckAnimatorManager.kt

```diff
@@ -117,9 +117,10 @@ internal class PuckAnimatorManager(
       accuracyCircleColor = settings2.accuracyRingColor
       accuracyCircleBorderColor = settings2.accuracyRingBorderColor
     }
-    bearingAnimator.apply {
-      enabled = settings2.puckBearingEnabled
-    }
+  }
+
+  internal fun setEnabled(enabled: Boolean) {
```

Comment:
`setEnabled` naming for the method that controls only bearing animator sounds a bit strange. Perhaps rename?

Reply:
Do you prefer `setPuckAnimationEnabled` (in next commit)?